docs/en/interfaces/formats.md

```diff
@@ -132,136 +132,7 @@ This format is also available under the names `TSVRawWithNamesAndNames`, `RawWit
 
 ## Template {#format-template}
 
-This format allows specifying a custom format string with placeholders for values with a specified escaping rule.
-
-It uses settings `format_template_resultset`, `format_template_row` (`format_template_row_format`), `format_template_rows_between_delimiter` and some settings of other formats (e.g. `output_format_json_quote_64bit_integers` when using `JSON` escaping, see further)
-
-Setting `format_template_row` specifies the path to the file containing format strings for rows with the following syntax:
-
-`delimiter_1${column_1:serializeAs_1}delimiter_2${column_2:serializeAs_2} ... delimiter_N`,
-
-where `delimiter_i` is a delimiter between values (`$` symbol can be escaped as `$$`),
-`column_i` is a name or index of a column whose values are to be selected or inserted (if empty, then column will be skipped),
-`serializeAs_i` is an escaping rule for the column values. The following escaping rules are supported:
-
-- `CSV`, `JSON`, `XML` (similar to the formats of the same names)
-- `Escaped` (similar to `TSV`)
-- `Quoted` (similar to `Values`)
-- `Raw` (without escaping, similar to `TSVRaw`)
-- `None` (no escaping rule, see further)
-
-If an escaping rule is omitted, then `None` will be used. `XML` is suitable only for output.
-
-So, for the following format string:
-
-      `Search phrase: ${SearchPhrase:Quoted}, count: ${c:Escaped}, ad price: $$${price:JSON};`
-
-the values of `SearchPhrase`, `c` and `price` columns, which are escaped as `Quoted`, `Escaped` and `JSON` will be printed (for select) or will be expected (for insert) between `Search phrase:`, `, count:`, `, ad price: $` and `;` delimiters respectively. For example:
-
-`Search phrase: 'bathroom interior design', count: 2166, ad price: $3;`
-
-In cases where it is challenging or not possible to deploy format output configuration for the template format to a directory on all nodes in a cluster, or if the format is trivial then `format_template_row_format` can be used to set the template string directly in the query, rather than a path to the file which contains it.
-
-The `format_template_rows_between_delimiter` setting specifies the delimiter between rows, which is printed (or expected) after every row except the last one (`\n` by default)
-
-Setting `format_template_resultset` specifies the path to the file, which contains a format string for resultset. Setting `format_template_resultset_format` can be used to set the template string for the result set directly in the query itself. Format string for resultset has the same syntax as a format string for row and allows to specify a prefix, a suffix and a way to print some additional information. It contains the following placeholders instead of column names:
-
-- `data` is the rows with data in `format_template_row` format, separated by `format_template_rows_between_delimiter`. This placeholder must be the first placeholder in the format string.
```

Comment:
I see that we have removed this section - don't we need to describe all these place holders for `format_template_resultset_format`?

Reply:
These descriptions are moved to line 96 under heading `format_template_resultset`. I thought to move `format_template_row_format`, `format_template_resultset_format` under heading `In-line specification` as they work the same as their respective settings with out `_format` except you specify the format string in-line rather than as a path to a file containing it. I will add a note that the rules for `format_template_row` and `format_template_resultset` still apply for the in-line variants.